runtime/shared_common/OSCache.cpp

```diff
@@ -210,7 +210,32 @@ SH_OSCache::getCacheDir(J9JavaVM* vm, const char* ctrlDirName, char* buffer, UDA
 #endif /*defined(OPENJ9_BUILD) */
 
 	rc = j9shmem_getDir(ctrlDirName, flags, buffer, bufferSize);
-	if (rc == -1) {
+
+	if (rc < 0) {
+		if (0 != vm->sharedCacheAPI->verboseFlags) {
```

Comment:
Seems this code could be in a helper function, so it's not duplicated twice.

Reply:
Changed the code to call new helper function j9shr_getCacheDir().